drake/systems/plants/collision/ResultCollector.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+#include "drake/systems/plants/collision/DrakeCollision.h"
+#include "ResultCollector.h"
+
+using namespace std;
+using namespace Eigen;
+
+namespace DrakeCollision {
+Vector3d toVector3d(const Vector3d& vec) { return vec; }
```

Comment:
I don't understand why this function has to be moved -- it appears only to be called from BulletResultCollector.cpp.